.github/workflows/std-workflows.yml

```diff
@@ -0,0 +1,34 @@
+name: GitHub Actions CI/CD
+run-name: Deploying Local CI.
+on:
+  push:
+    branches:
+      - exercise-04
```

Comment:
I think should be changed to match all the branches. 
Remember, unless the action is present in the branch, it won't be detected by it.

Reply:
It's mean this should me trigger all the branches ?

Reply:
yes, The goal is to run CI for all branches? unless you have any other strategy